samples/language/Operations.qs

```diff
@@ -0,0 +1,25 @@
+/// # Sample
+/// Operations   
+///
+/// # Description
+/// Operations are the basic building blocks of a Q# program. A Q#
+/// operation is a quantum subroutine. That is, it's a callable routine
+/// that contains quantum operations that modify the state of the
+/// qubit register.
+namespace MyQuantumApp {
+	
+    @EntryPoint()
+    operation MeasureOneQubit() : Result {
+        // Allocate a qubit, by default it is in zero state      
+        use q = Qubit();  
+        // We apply a Hadamard operation H to the state
+        // It now has a 50% chance of being measured 0 or 1  
```

Comment:
```suggestion
        // We apply a Hadamard transformation to qubit `q` using the H operation
```

Reply:
Removed the second line of the comment since it is not very relevant for the purposes of this specific sample.